docs/onboarding.md

```diff
@@ -28,21 +28,35 @@ You will see a pop-up in which you can insert your product name and choose
 between different product types:
 
 * **Syndicated Product** to start onboarding a new product in
-  [Syndication](syndication.md) mode
+  [Syndication](syndication.md) mode. This option is typically enabled by default
+  on any environment.
 * **Docker Application** to start onboarding a new product using one or more
-  [Docker image](docker.md)
+  [Docker image](docker.md). This option is disabled by default, and can be
+  enabled on specific environments.
 * **Service Product** to start [onboarding a new service](service.md), that is
-  not tied to the distribution of an application to the end-user
-* **API Product** to onboard a new [API-based service](api-product.md)
+  not tied to the distribution of an application to the end-user. This option
+  is disabled by default, and can be enabled for specific environments.
+* **API Product** to onboard a new [API-based service](api-product.md). This
+  option is disabled by default, and can be enabled by activating the
+  [Kong API Management](https://konghq.com/) connector.
```

Comment:
io non starei a menzionare kong qui, l'avrai già scritto nel primo paragrafo della pagina dedicata, altrimenti ripeti le stesse informazioni in posti diversi

Reply:
ok, ho approfittato anche per _riaggiustare_ il testo